### PR TITLE
Added pronouns for IDs

### PR DESCRIPTION
**Pull request by distributive**
Draft IDs weren't specified in the list I was given so they have been excluded for now

### CODE REVIEW

File: package-lock.json
Comment (plural) [inline package-lock.json:1]: does this change need to be in this PR?

File: schema/v1/card_schema.json
Comment (plural) [inline schema/v1/card_schema.json:1]: I don't believe you need this file.

File: v2/cards/khan_savvy_skiptracer.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "influence_limit": 12,
   "is_unique": false,
   "minimum_deck_size": 40,
+  "pronouns": "he/him",
```
Comment (plural) [inline v2/cards/khan_savvy_skiptracer.json:11]: is this one correct?

File: v2/cards/apex_invasive_predator.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
   "influence_limit": 25,
   "is_unique": false,
   "minimum_deck_size": 45,
+  "pronouns": "[missing]",
```
Comment (NoahTheDuke) [inline v2/cards/apex_invasive_predator.json:11]: it/its, right?

**Comment by NoahTheDuke** [comment]
I think keeping the non-NSG pronouns in is good. (Quetzal, Nero, etc.) Is it just "we don't own those" or is there another reason?

**Comment by distributive** [comment]
It was a request by Narrative

**Review by plural — APPROVED** [review]
Nice, thanks!!!

**Pull request merged**